lib/cmock_header_parser.rb

```diff
@@ -41,14 +44,60 @@ def parse(name, source)
       end
     end
 
+    if @treat_inlines == :include
```

Comment:
a slight improvement, since you're always using `:normalized_source` below:

```
@normalized_source = if (@treat_inlines == :include)
  transform_inline_functions(source)
else
  ''
end
```

I hope I'm not digging into too many minor details. You mentioned you're not strong at Ruby, so I thought you might like a little best-practices along the way. :)

With the way I've put it above, `@normalized_source` will always be defined to be something, just in case it's ever directly addressed (which it is just a little further down).

Reply:
Great! Thanks for taking the time to list these best-practices. By all means, keep them coming :)